src/dropdown.ts

```diff
@@ -299,55 +308,54 @@ export default class Dropdown extends LitElement implements FormControl {
   }
 
   set value(value: string[]) {
-    // You'll notice `this.#value` is internally often used directly. That's done to
-    // bypass the setter's side effects to either avoid a bad state or a prevent a loop.
-    //
-    // If `#onOptionsSelectedChange()`, for example, weren't to bypass the setter, then
-    // our setting value below would cause `#onOptionsSelectedChange()` to be called
-    // again, which in turn would call this setter.
-    this.#value = value;
-
     if (!this.multiple && value.length > 1) {
       throw new Error('Only one value is allowed when not `multiple`.');
     }
 
+    this.#value = value;
```

Comment:
A slight difference in behavior setting this after throwing. Happy to add a changeset if you like. But I thought it was insignificant enough to omit from an already long set of changes.